Frontend/src/pages/events/index.jsx

```diff
@@ -16,6 +16,15 @@ import { CompetitionContext } from '../../api/helper/context/competition_context
 import { setMessage } from '../../ui/events/messages'
 import LoadingMessage from '../../ui/messages/loadingMessage'
 
+function centiSecondsToHumanReadable(centiSeconds) {
```

Comment:
I feel like we have a similar function over in the Monolith already. Did you copy this method from over there?

Reply:
There's two functions. 
https://github.com/thewca/worldcubeassociation.org/blob/d3338d0175c608eb34bc1d3be60034cf0ac44e46/WcaOnRails/app/webpacker/lib/wca-live/attempts.js#L150
and https://github.com/thewca/worldcubeassociation.org/blob/d3338d0175c608eb34bc1d3be60034cf0ac44e46/WcaOnRails/app/webpacker/lib/utils/edit-events.js#L75
I'll see if these are correct

Reply:
`centisecondsToClockFormat` does not include any humanized strings like `"seconds"` and instead just returns the raw parsed time. You probably want to go with the one from `edit-events`.

Sidenote: Do we have a plan in mind for sharing frontend logic? Further down the line, there will be many more cases of "stealing" JS code from the monolith